keps/sig-windows/0000-20190103-windows-node-support.md

```diff
@@ -2,8 +2,9 @@
 kep-number: 0
 title: Windows node support
 authors:
-  - "@benmoss"
   - "@astrieanna"
+  - "@benmoss"
+  - "@patricklang"
 owning-sig: sig-windows
 participating-sigs:
   - sig-architecture
```

Comment:
as per the discussion today, sig-testing and sig-release should be reviewers for this part of the KEP